src/components/global/BreadcrumbTitle.vue

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+/**
+ * BreadcrumbTitle Component
+ *
+ * @description * Use this component to display page title as a part of
+ * a breadcrumb.
+ *
+ * @props
+ * - `title` (string, required): The object representing title of the
+ * current page.
+ *   It should be of type `string`.
+ *
+ * @example
+ * <breadcrumb-title :title="$t('index.titleHome')" />
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=4858-105197&t=UxaDEZYDQu3Og9i6-1)
+ */
+
+// libraries
+import { computed, defineComponent } from 'vue';
+import { useRoute } from 'vue-router';
+
+// types
+import type { BreadcrumbPage } from '../types/Breadcrumb';
+
+export default defineComponent({
+  name: 'BreadcrumbTitle',
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+  },
+  setup() {
+    const route = useRoute();
+
+    const pages = computed((): BreadcrumbPage[] => {
+      // get meta object
+      if (!route?.matched.length || !route.matched[0]?.meta?.breadcrumb) {
+        return [];
+      }
+      const breadcrumb = route.matched[0].meta.breadcrumb;
+      if (!Array.isArray(breadcrumb)) {
```

Comment:
If condition in one line ` if (!Array.isArray(breadcrumb)) return[];`

Reply:
Updated.